src/tiles/grid/styles/tile-grid-shared-styles.ts

```diff
@@ -0,0 +1,98 @@
+import { css } from 'lit';
```

Comment:
this shared style being used for tile system (item/patron/collection/etc...) only from now.

Reply:
Nice! I like that there's a single source of truth for some of these shared tile styles now.